identity/handler.go

```diff
@@ -597,13 +597,21 @@ func (h *Handler) patch(w http.ResponseWriter, r *http.Request, ps httprouter.Pa
 		return
 	}
 
+	credentials := identity.Credentials
+
 	oldState := identity.State
 
 	if err := jsonx.ApplyJSONPatch(requestBody, identity, "/id", "/stateChangedAt", "/credentials"); err != nil {
 		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrBadRequest.WithReasonf("%s", err).WithWrap(err)))
 		return
 	}
 
+	// See https://github.com/ory/cloud/issues/148
+	// The apply patch operation overrides the credentials with an empty map.
+	if len(identity.Credentials) == 0 {
```

Comment:
It probably makes sense to do that always, given that patch ignores the credentials key. WDYT?

Reply:
Hm, sure. I wanted to account for future situations, where we allow that, but I guess in that case you'd have to touch this place anyway. So doesn't matter! Will change it.

Reply:
When in doubt, leave it out :)